releasenotes/notes/3399-Change-QuantumCircuit-method-input-arg-names-8422b6abfdfff081.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    Methods for adding gates to a `QuantumCircuit` with abbreviated keyword
+    arguments (e.g. `ctl`, `tgt`) have had their keyword arguments renamed
+    to be more descriptive (e.g. `control_qubit`, `target_qubit`). The old
+    names have been deprecated.
```

Comment:
It'd probably be good to have a table in here showing the old names and the new ones.

Reply:
:+1: Added in f6d10c1472300e3a0d98e46ddd3df53212bc8ff7.